packages/merchant-tablet-ionic/angular.json

```diff
@@ -1,5 +1,5 @@
 {
-	"$schema": "../../node_modules/@angular/cli/lib/config/schema.json",
+	"$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
@YavorGrancharov why you changed that line too? Was it wrong before? I.e. path started from "../../node_modules..." before and I thought it was correct or?

Reply:
It is with yellow underline and gives this message : Unable to load schema from 'c:\ever\node_modules\@angular\cli\lib\config\schema.json': ENOENT: no such file or directory, open 'c:\ever\node_modules\@angular\cli\lib\config\schema.json'.(768). The reference is pointing to global node_modules where such folder does not exist, but do exists
 in local node_modules of the merchant app, so I changed the refreence path.
![ADNcGfsT5O](https://user-images.githubusercontent.com/28997304/82286882-78976f80-99a7-11ea-9f08-254152651637.png)

Reply:
@YavorGrancharov hm, that's really weird as all other packages have same path in angular.json files, so I wonder what is the difference and should we change it there too?? How do you get that error you described? Is it during `ng build` run or when?

Reply:
I cloned the repo again and the filepath in angular.json was giving me same error, beacause node_modules were not present. After yarn bootstrap it gives me same error and when I go to currently installed global node_modules I can't find any folder with name @angular where the schema should be.The full path to folder is @angular/cli/lib/config/schema.json and it is present only in local node_modules of each app . The error persists after yarn build:all too!